src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -710,9 +722,15 @@ bool CorProfiler::IsAttached() const { return is_attached_; }
 //
 // Startup methods
 //
-HRESULT CorProfiler::RunILStartupHook(const ComPtr<IMetaDataEmit2>& metadata_emit,
-                                      const ModuleID module_id,
-                                      const mdToken function_token) {
+bool CorProfiler::ProfilerIsLoadedIntoAppDomain(AppDomainID app_domain_id) {
```

Comment:
Consider renaming this method to `ProfilerAssemblyIsLoadedIntoAppDomain()`. The way I see it, the "profiler" is the native library (`Datadog.Trace.ClrProfiler.Native.dll`/`.so`) and it's only loaded once per process. The profiler assembly is the `Datadog.Trace.ClrProfiler.Managed.dll` file , which can be loaded into multiple AppDomains.

Reply:
Thanks, fixed in 10d1bec586887580397d6506bf090d9251c31c48 .